scripts/scil_score_tractogram.py

```diff
@@ -417,86 +436,122 @@ def compute_vb_vs_all_bundles(
      +
     WPC connections:
        1) connect the head and tail but criteria 2 and 3 are not respected
+
+
+    Returns
+    -------
+    vb_sft_list: list
+        List of StatefulTractograms of VS
```

Comment:
Maybe add points at the end of each sentence?

Reply:
Dont need it for parameters and returns. Only for the help.